lib/modules/platform/gerrit/index.ts

```diff
@@ -0,0 +1,553 @@
+ */
+export async function setBranchStatus(
+  branchStatusConfig: BranchStatusConfig
+): Promise<void> {
+  const { labelName, label } = mapBranchStateContextToLabel(
+    branchStatusConfig.context
+  );
+  const labelValue =
+    label && mapBranchStatusToLabel(branchStatusConfig.state, label);
+  if (labelName && labelValue) {
+    const pr = await getBranchPr(branchStatusConfig.branchName);
+    if (pr === null) {
+      return Promise.resolve();
+    }
+    await client.setLabel(pr.number, labelName, labelValue);
+  }
+  return Promise.resolve();
+}
+
+function mapBranchStateContextToLabel(context: string | null | undefined): {
+  labelName?: string;
+  label?: GerritLabelTypeInfo;
+} {
+  let labelName;
+  switch (context) {
+    case 'renovate/stability-days':
+      labelName = config.labelMappings?.stabilityDaysLabel;
+      break;
+    case 'renovate/merge-confidence':
+      labelName = config.labelMappings?.mergeConfidenceLabel;
+      break;
+  }
+  if (labelName && config.labels[labelName]) {
+    return {
+      labelName,
+      label: config.labels[labelName],
+    };
+  }
+  return {};
+}
+
+export function getRawFile(
+  fileName: string,
+  repoName?: string,
+  branchOrTag?: string
+): Promise<string | null> {
+  const repo = repoName ?? config.repository ?? 'All-Projects';
+  const branch = branchOrTag ?? config.head ?? 'HEAD';
+  return client.getFile(repo, branch, fileName);
+}
+
+export async function getJsonFile(
+  fileName: string,
+  repoName?: string,
+  branchOrTag?: string
+): Promise<any | null> {
+  const raw = (await getRawFile(fileName, repoName, branchOrTag)) as string;
+  return JSON5.parse(raw);
+}
+
+export function getRepoForceRebase(): Promise<boolean> {
+  return Promise.resolve(false);
+}
+
+export async function addReviewers(
+  number: number,
+  reviewers: string[]
+): Promise<void> {
+  for (const reviewer of reviewers) {
+    await client.addReviewer(number, reviewer);
+  }
+}
+
+/**
+ * add "CC" (only one possible)
+ */
+export async function addAssignees(
+  number: number,
+  assignees: string[]
+): Promise<void> {
+  if (assignees.length) {
+    await client.addAssignee(number, assignees[0]);
+  }
+}
+
+export async function ensureComment(
+  ensureComment: EnsureCommentConfig
+): Promise<boolean> {
+  logger.debug(
+    `ensureComment(${ensureComment.number}, ${ensureComment.topic!}, ${
+      ensureComment.content
+    })`
+  );
+  const commentExists = await checkForExistingMessage(
+    ensureComment.number,
+    ensureComment.content,
+    ensureComment.topic
+  );
+  if (commentExists) {
+    return true;
+  }
+  await client.addMessage(
+    ensureComment.number,
+    ensureComment.content,
+    ensureComment.topic ?? undefined
+  );
+  return true;
+}
+
+export function massageMarkdown(prBody: string): string {
+  //TODO: do more Gerrit specific replacements?
+  return smartTruncate(smartLinks(prBody), 16384) //TODO: check the real gerrit limit (max. chars)
```

Comment:
Does gerrit support checkboxes? otherwise use the `readOnlyIssueBody` from platform utils